src/pages/MyAccount.tsx

```diff
@@ -0,0 +1,66 @@
+import styled from '@emotion/styled';
+import { useContext, useEffect } from 'react';
+import { useNavigate } from 'react-router-dom';
+
+import { authContext } from '@/context/AuthContext';
+
+const MyAccountContainer = styled.div`
+  width: 100%;
+  padding: 80px 0px 120px;
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  justify-content: center;
+  font-weight: 700;
+  font-size: 36px;
+`;
+
+const DivHeight = styled.div`
+  width: 100%;
+  height: 64px;
+`;
+
+const StyledButton = styled.button`
+  max-width: 200px;
+  width: 100%;
+  border-radius: 4px;
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  cursor: pointer;
+  transition: background-color 200ms ease 0s;
+  height: 40px;
+  font-size: 15px;
+  color: rgb(255, 255, 255);
+  background-color: rgb(68, 68, 68);
+
+  &:hover {
+    background-color: rgb(85, 85, 85);
+  }
+`;
+
+const MyAccount = () => {
+  const nav = useNavigate();
```

Comment:
```suggestion
  const navigate = useNavigate();
```

마이너하지만 이런 건 안줄여도 되지 않을까 싶긴 하네요 ㅎㅎ
명확함이 항상 더 중요한 가치인 것 같아요